src/middleware/errorHandler.ts

```diff
@@ -1,20 +1,16 @@
 import { ErrorRequestHandler } from "express";
-import { UnauthorizedError } from "express-oauth2-jwt-bearer";
+
```

Comment:
is this in purpose?

Reply:
yes, I changed the jwt validation, got rid of this unmaintained library